### PR TITLE
Remove matrix parameters, addresses #159.

### PR DESCRIPTION
**Pull request by peacekeeper**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/285.html" title="Last updated on May 14, 2020, 8:10 PM UTC (a5f81c6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/285/48bc712...a5f81c6.html" title="Last updated on May 14, 2020, 8:10 PM UTC (a5f81c6)">Diff</a>

### CODE REVIEW

**Comment by peacekeeper** [comment]
As discussed in #159, and based on the resolution in the 2020-04-07 WG meeting (see [minutes](https://www.w3.org/2019/did-wg/Meetings/Minutes/2020-04-07-did#section5)).

File: index.html [hunk 1]
```diff
@@ -1052,7 +1058,7 @@ <h2>Method-Specific DID URL Parameters</h2>
         </p>
 
         <pre class="example nohighlight">
-did:example:21tDAKCERh95uGgKbJNHYp;service=agent;foo:bar=high
+did:example:21tDAKCERh95uGgKbJNHYp?service=agent&foo:bar=high
```
Comment (OR13) [inline index.html:1061]: prefer not to see foo:bar as the example here if there is _any_ better option.

Comment (peacekeeper) [reply index.html:1061]: Thanks @OR13 , I think this is orthogonal to the matrix/query parameter change and should be discussed in another issue (e.g. in https://github.com/w3c/did-core/issues/36 or a new one).

Comment (OR13) [reply index.html:1061]: yes I agree. you can resolve this.

**Comment by msporny** [comment]
Normative, debated heavily with consensus reached, multiple positive reviews, merging.

**Pull request merged**